### ISSUE TITLE
default browser coutermeasure doesn't crash the browser..

### ISSUE THREAD

**Issue by mini-programmer** [MichaelXF/js-confuser#102]
I'm not really sure if this should be a feature request or issue
When it's supposed to "crash the process" on browser, all it does it run `document.documentElement.innerHTML = '';` which doesn't crash anything
maybe it could run an infinite loop to actually crash? like `for(;;){while(1){}}`

**Comment by MichaelXF** [comment]
You are right. The new behavior will just be an infinite loop as the default countermeasure. I will change this for next update.

**Comment by Le0Developer** [comment]
An infinite loop can be easily stopped by just pressing the "Pause script execution" button in devtools.
There are better ways to crash a tab like spamming iframes. (this absolutely kills devtools if it runs for a few seconds)
```js
for(;;) {
    let a = document.createElement("iframe");
    a.src = "/"
    document.body.append(a)
}
```

**Comment by fuzzbuck** [comment]
Maybe the countermeasure should be configurable?

Some may want to just pause script execution, and others crash the page..

**Comment by MichaelXF** [comment]
@fuzzbuck The [countermeasures](https://github.com/MichaelXF/js-confuser/blob/master/docs/Countermeasures.md) is configurable by setting it a string, it can be a custom callback. This issue is related to the default behavior (no value provided) is incorrect. I like @Le0Developer 's code snippet and it will be the default behavior
